.github/workflows/alpha-release.yml

```diff
@@ -16,77 +16,82 @@ jobs:
       changelog: ${{ steps.changelog.outputs.changelog }}
       release_id: ${{ steps.release.outputs.id }}
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4.1.3
```

Comment:
Can't we just specify the major version?

Reply:
When checking more I see that you have specified patch release version numbers for all actions. Is you idea to make the build more stable and not risk breakage if any of the actions change? Otherwise I think the common usage is to only specify major version for GHA actions.

Reply:
I did this at night, and just completely forgot about the fact that we can use major versions only. Pushed a patch.